odbc/src/api/definitions.rs

```diff
@@ -153,43 +153,48 @@ pub enum EnvironmentAttribute {
 #[repr(i32)]
 #[derive(Debug, PartialEq, Eq, Clone, Copy, FromPrimitive)]
 pub enum StatementAttribute {
-    SQL_ATTR_APP_ROW_DESC = 10010,
-    SQL_ATTR_APP_PARAM_DESC = 10011,
-    SQL_ATTR_IMP_ROW_DESC = 10012,
-    SQL_ATTR_IMP_PARAM_DESC = 10013,
     SQL_ATTR_CURSOR_SCROLLABLE = -1,
     SQL_ATTR_CURSOR_SENSITIVITY = -2,
```

Comment:
I reordered these by number to make it easier to see what we were missing.

Reply:
Good idea. I wasn't sure why it was in this order in odbc_sys in the first place! It was helpful during #64 to keep this order since it's the same order we used in the `functions.rs` `match` statement, but switching it to numeric order is super useful 🧠